src/components/sync-connected-sites.tsx

```diff
@@ -196,10 +202,32 @@ type SyncConnectedSitesListProps = {
 	selectedSite: SiteDetails;
 };
 
+const getStatusWithProgress = (
+	__: I18n[ '__' ],
+	sitePullState?: PullStateProgressInfo,
+	importState?: ImportProgressState[ string ]
+) => {
+	if ( ! importState && sitePullState ) {
+		return { message: sitePullState.message, progress: sitePullState.progress };
+	}
+	if ( importState ) {
+		if ( importState.progress === 100 ) {
+			return { message: __( 'Applying final details' ), progress: 99 };
```

Comment:
Let's also add ellipsis to this sentence:
```suggestion
			return { message: __( 'Applying final details…' ), progress: 99 };
```